staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -527,6 +527,10 @@ func updateServiceLoadBalancerIP(service *v1.Service, serviceIP string) *v1.Serv
 }
 
 func (az *Cloud) findServiceIPAddress(ctx context.Context, clusterName string, service *v1.Service, isInternalLb bool) (string, error) {
+	if len(service.Status.LoadBalancer.Ingress) > 0 && len(service.Status.LoadBalancer.Ingress[0].IP) > 0 {
```

Comment:
I think these lines should be moved after L536, in case the `spec.LoadBalancerIP` is updated from empty value.